### ISSUE TITLE
Just FYI you broke Crusade detachments with your latest update, definitely something to test before pushing a build!

### ISSUE THREAD

**Issue by chronocobra** [BSData/horus-heresy#1979]
Think you applied the rules of "Varangian Heresy" to everything.

**Comment by Mayegelt** [comment]
> Think you applied the rules of "Varangian Heresy" to everything.

How so? What is broken?
The Varangian stuff shouldn't be affecting anything unless you use the Reltek FoC. Especially as no units are in it.
Please post up what issues you are having and I'll try to resolve them tonight with an emergency patch.

**Comment by Mayegelt** [comment]
I've checked myself and no issues on Legiones in Crusade at a quick first glance.

If you can post what errors you are getting and a list including RoW and Legions that would be good.

I know some issues will pop up in current lists such as now needed to tick a box in Chaplin's and Vigilalators on existing lists. This is due to some RoW requirements for Salamanders and Alpha Legion using a particular RoW.
The options to fix that are to either add a new category entry for them, or add a tick box selection entry.
Adding the tick box will add a single click to all old lists to fix and not a large amount of data.
Adding s Category adds a binary switch on every single selection entry (unit, weapon, sub option) which substantially increases the file size and slows down the whole file, making it less usable on older devices.

**Comment by Mayegelt** [comment]
Noticed 1 unexpected issue. On my side of the files on Desktop it was working fine, but it seems to have reversed a switch on uploading and transferring for the Heavy Support restriction. Hopefully that was the issue you were experiencing, and will correct that now.
There is an issue for the data editor that due to the sheer size of the file it seems to on occasion switch a 0 to a 1 rather randomly. This one was caused by making the Dark Brethren RoW to work. I had tested it my end on PC and it had worked fine, but then at some point it seemed to have switched.

There has also been several updates in a small amount of time, so hopefully you are on the latest one, as a few issues were noticed with each push. Most of which were unrelated to the issues being fixed at the time.
